src/__tests__/lib/containers/MarkdownSynapse.test.tsx

```diff
@@ -1,30 +1,79 @@
 import * as React from 'react'
-import { shallow } from 'enzyme'
+import { mount, shallow } from 'enzyme'
 import MarkdownSynapse, {
   MarkdownSynapseProps,
 } from '../../../lib/containers/MarkdownSynapse'
 import Bookmarks from '../../../lib/containers/widgets/Bookmarks'
 import SynapseImage from '../../../lib/containers/widgets/SynapseImage'
 import SynapseVideo from '../../../lib/containers/widgets/SynapseVideo'
-import SynapsePlot from '../../../lib/containers/widgets/SynapsePlot'
 import { delay } from '../../../lib/utils/SynapseClient'
 import { _TIME_DELAY } from '../../../lib/utils/SynapseConstants'
+import {
+  MOCK_CONTEXT,
+  SynapseTestContext,
+} from '../../../mocks/MockSynapseContext'
+import {
+  SynapseContextProvider,
+  SynapseContextType,
+  withSynapseContextConsumer,
+} from '../../../lib/utils/SynapseContext'
+import { render, screen } from '@testing-library/react'
+
+jest.mock('../../../lib/containers/widgets/SynapsePlot', () => {
+  return {
+    __esModule: true,
+    default: function MockSynapsePlot() {
+      return <div role="figure"></div>
+    },
+  }
+})
+
+jest.mock('../../../lib/containers/widgets/SynapseImage', () => {
+  return {
+    __esModule: true,
+    default: function MockSynapseImage() {
+      return <div role="img"></div>
+    },
+  }
+})
```

Comment:
Simple example for how we can use mocks to reduce testing surface while using RTL's `render` or Enzyme's `mount`